fe-vite-react/src/SingleArticlePage.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { getSingleArticle, getArticleComments, formatCommentDate } from "./utils";
+
+const SingleArticlePage = () => {
+  const { article_id } = useParams();
+  const [comments, setComments] = useState([]);
+  const [singleArticle, setSingleArticle] = useState({});
+
+  useEffect(() => {
+    getSingleArticle(article_id)
+      .then((article) => {
+        setSingleArticle(article);
+      })
+      .catch((error) => {
+      });
+    getArticleComments(article_id)
+      .then((comments) => {
+        setComments(comments);
+      })
+      .catch((error) => {
+      });
+  }, [article_id]);
+
+  return (
+    <section className="single-article">
+      <div className="single-article-title">
+        <h2>{singleArticle.title}</h2>
+      </div>
+
+      <p>votes: {singleArticle.votes}</p>
+      <div className="single-article-img">
+        <img src={singleArticle.article_img_url} alt="picture of" />
+      </div>
+
+      <div className="single-article-content">
+        <p> By {singleArticle.author}</p>
+        <p>topic {singleArticle.topic}</p>
+        <div className="article-body">{singleArticle.body}</div>
+      </div>
```

Comment:
this could be its own component

Also I'd try and consider using more semantic tags than divs wherever possible.